core/model/src/main/java/org/eclipse/sw360/antenna/api/workflow/WorkflowStepResult.java

```diff
@@ -63,6 +64,14 @@ public boolean addAdditionalReportComment(String comment) {
         return additionalReportComments.add(comment);
     }
 
+    public Optional<Map.Entry<String, String>> getFailCausingResult() {
+        return this.failCausingResult;
+    }
+
+    public void addFailCausingResult(String configurableWorkflowItemName, String failCausingResult) {
```

Comment:
one could alternatively think about adding the failure as a fact to the artifact